### PR TITLE
feat(register): add register page

### LINKED ISSUE #3: Create register page

### PR DESCRIPTION
**Pull request by billalxcode**

### CODE REVIEW

File: lib/constants.dart
Comment (Hasban-Fardani) [inline lib/constants.dart:1]: base url does not start with /api, wait for me to release the documentation.
please change the name with camelCase

Comment (billalxcode) [reply lib/constants.dart:1]: camel case is not supported by code formatters

File: lib/components/input.dart
Comment (Hasban-Fardani) [inline lib/components/input.dart:1]: why errorBorder, enabledBorder, focusedBorder, and focusedErrorBorder has same color (color: WappstikPalette.grey)?

Comment (billalxcode) [reply lib/components/input.dart:1]: When an error occurs in validation, the input style will change to the default style. I want the input style not to change to the default style, that's why I set it like that.

Comment (billalxcode) [reply lib/components/input.dart:1]: And also when an error occurs, an error message will automatically appear below the input.

Comment (billalxcode) [reply lib/components/input.dart:1]: Like this

image

Comment (Hasban-Fardani) [reply lib/components/input.dart:1]: ahh i see. okay

**Review by Hasban-Fardani — APPROVED** [review]

**Pull request merged**